angular/src/components/verify-master-password.component.html

```diff
@@ -0,0 +1,17 @@
+<ng-container *ngIf="!usesKeyConnector">
+    <label for="masterPassword">{{'masterPass' | i18n}}</label>
+    <input id="masterPassword" type="password" name="MasterPasswordHash" class="form-control"
+        [formControl]="secret" required appAutofocus appInputVerbatim>
+</ng-container>
+<ng-container *ngIf="usesKeyConnector">
+    <div class="form-group">
+        <input type="button" class="btn btn-primary" value="Request one-time password" (click)="requestOtp()"
```

Comment:
* button text needs to be i18n'd
* not sure about `input type="button"`, why not just `<button>` ?

Reply:
Done.